test/snowpack/package/@nivo/index.test.js

```diff
@@ -0,0 +1,36 @@
+const dedent = require('dedent');
+const {testFixture, testRuntimeFixture} = require('../../../fixture-utils');
+const {fmtjson} = require('../../../test-utils');
+
+const pkg = {
+  'index.js': dedent`
+    import {NetworkCanvas} from '@nivo/network';
+  `,
+  'package.json': fmtjson({
+    dependencies: {
+      '@nivo/core': '^0.72.0',
+      '@nivo/network': '^0.72.0',
+    },
+  }),
+};
+
+/**
+ * Fixes #3466
+ * Main thing we’re testing for here is that dev server doesn’t hang on circular deps
+ * Though this test is slow, it’s important to test on real npm packages and not mocked ones
+ * as symlink behavior is really different here
+ */
+describe.skip('@nivo/core', () => {
```

Comment:
As the comment said, added a test here which can be run manually. But skipping this in GitHub because for some reason this test keeps flaking in GitHub CI. But I’m able to run it on my machine in Node 12, 14, and 16 without fail every time. So this does fix the issue!